tests/Node/IncludeTest.php

```diff
@@ -80,9 +80,10 @@ public static function provideTests(): iterable
 // line 1
 \$__internal_%s = null;
 try {
-    \$__internal_%s =     \$this->loadTemplate("foo.twig", null, 1);
+    \$__internal_%s = \$this->loadTemplate("foo.twig", null, 1);
 } catch (LoaderError \$e) {
     // ignore missing template
+    \$__internal_%s = null;
```

Comment:
this seems useless as it is already assigned before the try/catch. Why do we need it ?

Reply:
The EmbedNode extends this one and there the template loading doesn’t fail. It fails when trying to load the parent. So we need to reset the variable here for this use case.

Reply:
Removed the assign before the try/catch here: 
- https://github.com/twigphp/Twig/pull/4346